client/cli/src/client_proxy.rs

```diff
@@ -329,6 +331,48 @@ impl ClientProxy {
         }
     }
 
+    /// Allow executing arbitrary script in the network.
+    pub fn allow_custom_script(
+        &mut self,
+        space_delim_strings: &[&str],
+        is_blocking: bool,
+    ) -> Result<()> {
+        ensure!(
+            space_delim_strings.len() == 1,
+            "Invalid number of arguments for setting publishing option"
+        );
+        match self.faucet_account {
+            Some(_) => self.association_transaction_with_local_faucet_account(
+                transaction_builder::encode_publishing_option_script(
+                    VMPublishingOption::CustomScripts,
+                ),
+                is_blocking,
+            ),
+            None => unimplemented!(),
+        }
+    }
+
+    /// Only allow executing predefined script in the network.
+    pub fn disallow_custom_script(
```

Comment:
Similar: `disable_custom_scripts`?

Also: should this take the whitelist as an argument and/or clarify the comment to explain what "predefined" means?